### PR TITLE
fix some spell error, eg:mapedFile -> mappedFile， relate to https://github.com/apache/rocketmq-site/pull/119

### PR DESCRIPTION
**Pull request by Hen1ng**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

fix some spell error, eg:maped -> mapped

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java [hunk 1]
```diff
@@ -84,7 +84,7 @@
     @ImportantField
     private int fileReservedTime = 72;
     // Flow control for ConsumeQueue
-    private int putMsgIndexHightWater = 600000;
```
Comment (vongosling) [inline store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java:87]: This field should be addressed in our doc. You know, if we do not have any comments, people would feel a  little quirk in these points.

**Review by vongosling — CHANGES_REQUESTED** [review]

File: store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java (hunk 1 above)
Comment (Hen1ng) [reply store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java:87]: Could you tell me which doc this field should be addressed？ I just change this field name，I could not find the field named "putMsgIndexHightWater" in any doc under the path rocketmq/docs. thx

Comment (Hen1ng) [reply store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java:87]: @ni-ze please help to review this pr

**Review by HScarb — APPROVED** [review]

**Comment by Hen1ng** [comment]
@vongosling 
I have done this work to  https://github.com/apache/rocketmq-site，  https://github.com/apache/rocketmq-site/pull/119， actually, I do not find the field putMsgIndexHightWater, but i fount some other spell error

File: store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java (hunk 1 above)
Comment (Hen1ng) [reply store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java:87]: @HScarb thx for the answer

**Review by lwclover — APPROVED** [review]

**Review by yuz10 — APPROVED** [review]

**Review by yuz10 — APPROVED** [review]

File: store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java (hunk 1 above)
Comment (caigy) [reply store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java:87]: @Hen1ng Fields in `MessageStoreConfig` may be set by reflection (you may find such usage by `org.apache.rocketmq.common.MixAll#properties2Object`), once you change the field names, users who have configured by the previous names will be affected, the values they've changed in property file will be 'restored' to default values. So changing these fields require careful consideration.

IMO, you'd better move this modification to another dedicated issue, and change the field name in a compatible way also with clear documents  addressing to it.

**Review by caigy — CHANGES_REQUESTED** [review]

**Comment by tsunghanjacktsai** [comment]
According to the statement from @caigy, I noticed that there might have a certain amount of spelling errors in our codebase. However, the contributors often focused on the changes for the current version, and ignore affecting the users of the previous versions. Therefore, I suggest that we could open an issue aiming for these types of spelling mistakes, and regulate developers to correct them under the rules.

**Pull request closed without merging**